src/services/edusharing/hooks/merlin.hooks.js

```diff
@@ -10,7 +18,7 @@ const validateReference = (hook) => {
 };
 
 exports.before = {
```

Comment:
maybe better warp the feature flag around the export.

Reply:
1. Ok, I changed it to `FEATURE_ES_MERLIN_ENABLED` in code and Travis
2. This seems to be the way to do it by the looks of how other features works in our application.
3. The service is accessed several places and should be a hook in before and on all requests.

Reply:
Okay, have you tested it lokal without merlin and show that no new error where add to the log?

Reply:
I have tested it. But I found a better solution. If the feature is not enabled, it has a fallback that returns the default merlin url instead. So either way, the user get's redirected somewhere.

Reply:
And what is if the default merlin URL are change or so ?

Reply:
Then the user would be redirected to the new default Merlin URL.